apps/emqx_management/src/emqx_mgmt_api_clients.erl

```diff
@@ -656,6 +673,30 @@ fields(message) ->
         {from_clientid, hoconsc:mk(binary(), #{desc => ?DESC(msg_from_clientid)})},
         {from_username, hoconsc:mk(binary(), #{desc => ?DESC(msg_from_username)})},
         {payload, hoconsc:mk(binary(), #{desc => ?DESC(msg_payload)})}
+    ];
+fields(requested_client_fields) ->
+    %% NOTE: some Client fields actually returned in response are missing in schema:
+    %%  enable_authn, is_persistent, listener, peerport
```

Comment:
Should these fields be included to `client` schema? 
BTW, `peerport` is redundant as there is simply `port`.

Reply:
yes, make sense to add to schema

Reply:
maybe remove `port`